AUTHORS

```diff
@@ -100,4 +100,5 @@ Jingyi Chen <jingyichen@link.cuhk.edu.cn>
 Junhua Lin <1075209054@qq.com>
 Tomasz Gucio <tgucio@gmail.com>
 Jason C.H <ctrysbita@outlook.com>
-Hubert Jóźwiak <hjozwiakdx@gmail.com>
\ No newline at end of file
+Hubert Jóźwiak <hjozwiakdx@gmail.com>
+David Neuy <quantjump@gmail.com>
```

Comment:
Nit: There should be a newline here at the end of the file

Reply:
@Piinks Added thee new line.
All checks pass now :smiley: